src/utilcode/util.cpp

```diff
@@ -957,7 +959,7 @@ DWORD LCM(DWORD u, DWORD v)
     CONTRACTL_END;
 
 #if !defined(FEATURE_REDHAWK) && (defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_))
-    BOOL enableGCCPUGroups     = CLRConfig::GetConfigValue(CLRConfig::EXTERNAL_GCCpuGroup) != 0;
+    BOOL enableGCCPUGroups     = Configuration::GetKnobBooleanValue(W("System.GC.CpuGroup"), CLRConfig::EXTERNAL_GCCpuGroup);
```

Comment:
This is adding a new official `.runtime.json` switch. Is it intentional?

Reply:
yes, we want to allow all public facing GC configs to be specifiable via runtimeconfig.json.

Reply:
- It may be useful to get the list of the new public facing configs and make sure that they are designed well and have good names.
- The new public facing GC configs are a new feature that should be approved for 3.0 by @MeiChin-Tsai at this point.